store/chain.ts

```diff
@@ -0,0 +1,25 @@
+import { GetterTree, ActionTree, MutationTree, Commit } from 'vuex'
+
+export const state = () => ({
+  chainProperties: {},
```

Comment:
may add type here ? 
there is just `ss58Format, tokenDecimals, unit`

Reply:
unit?